docs/setup_help/m1_mac_setup.md

```diff
@@ -0,0 +1,47 @@
+# Notes from Wes Goar on setup of vrs-python on a m1 mac
+- @author Wes Goar
```

Comment:
Is `@author` meaningful? I'm unaware of that tag.

Reply:
That was a carry-over from my days coding in R. I have removed it.